frontend/src/app.vue

```diff
@@ -63,18 +63,42 @@
 
 <script setup lang="ts">
 import { signOutManually } from './utils/auth';
-
+import { reloadNuxtApp } from 'nuxt/app';
 const $supabaseClient = useSupabaseClient();
-
 $supabaseClient.auth.onAuthStateChange((event) => {
   switch (event) {
     case 'SIGNED_OUT':
       signOutManually();
```

Comment:
this will remove, cookies, states etc... but in the Supabase the user session is still active. use `await signOut()` instead